api/v1alpha1/fenceagentsremediation_types.go

```diff
@@ -23,16 +23,39 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-type ParameterName string
-type NodeName string
-
 const (
 	// FARFinalizer is a finalizer for a FenceAgentsRemediation CR deletion
 	FARFinalizer string = "fence-agents-remediation.medik8s.io/far-finalizer"
 	// Taints
 	FARNoExecuteTaintKey = "medik8s.io/fence-agents-remediation"
+	// FenceAgentActionSucceededType is the condition type used to signal whether the Fence Agent action was succeeded successfully or not
+	FenceAgentActionSucceededType = "FenceAgentActionSucceeded"
+	// error status messages
```

Comment:
these errors are used for tests only, correct? They shouldn't be in the api package then.

Reply:
I have moved them to `errors.go` file